robocop/checkers/naming.py

```diff
@@ -215,6 +217,7 @@ class KeywordNamingChecker(VisitorChecker):
 
     def __init__(self):
         self.letter_pattern = re.compile(r"\W|_", re.UNICODE)
+        self.in_if_block = False
```

Comment:
Do you mean 'inside IF block'? If yes, then I would recommend renaming it to `inside_if_block`.